ixmp/backend/io.py

```diff
@@ -84,7 +88,23 @@ def s_write_excel(be, s, path, item_type):
                 empty_sets.append((name, data))
             continue
 
-        data.to_excel(writer, sheet_name=name, index=False)
+        if len(data) > max_row:
+            for i in range(1, int(np.ceil(len(data) / max_row)) + 1):
```

Comment:
Can you name I as sheet_num or something meaningful?

Reply:
Thanks, renamed to _sheet_num_